WordPress/Classes/ViewRelated/Blog/Blog List/BlogListViewController.m

```diff
@@ -1,1073 +0,0 @@
-    self.tableView.contentInset = UIEdgeInsetsMake(self.view.safeAreaInsets.top, insets.left, keyboardHeight, insets.right);
-}
-
-- (void)keyboardWillHide:(NSNotification*)notification
-{
-    self.tableView.scrollIndicatorInsets = UIEdgeInsetsZero;
-    self.tableView.contentInset = UIEdgeInsetsZero;
-}
-
--(CGRect)localKeyboardFrameFromNotification:(NSNotification *)notification
-{
-    CGRect keyboardFrame = [notification.userInfo[UIKeyboardFrameEndUserInfoKey] CGRectValue];
-
-    // Convert the frame from window coordinates
-    return [self.view convertRect:keyboardFrame fromView:nil];
-}
-
-- (void)wordPressComApiDidLogin:(NSNotification *)notification
-{
-    [self.tableView reloadSections:[NSIndexSet indexSetWithIndex:0] withRowAnimation:UITableViewRowAnimationFade];
-}
-
-- (void)wordPressComApiDidLogout:(NSNotification *)notification
-{
-    [self.tableView reloadSections:[NSIndexSet indexSetWithIndex:0] withRowAnimation:UITableViewRowAnimationFade];
-}
-
-- (void)wordPressComAccountChanged:(NSNotification *)notification
-{
-    [self.tableView reloadData];
-    [self setEditing:NO];
-    [self updateSearchVisibility];
-}
-
-#pragma mark - Table view delegate
-
-- (NSString *)tableView:(UITableView *)tableView titleForDeleteConfirmationButtonForRowAtIndexPath:(NSIndexPath *)indexPath
-{
-    return NSLocalizedString(@"Remove", @"Button label when removing a blog");
-}
-
-- (BOOL)tableView:(UITableView *)tableView canEditRowAtIndexPath:(NSIndexPath *)indexPath
-{
-    return YES;
-}
-
-- (UITableViewCellEditingStyle)tableView:(UITableView *)tableView editingStyleForRowAtIndexPath:(NSIndexPath *)indexPath
-{
-    if (self.tableView.isEditing) {
-        return UITableViewCellEditingStyleNone;
-    } else {
-        return UITableViewCellEditingStyleDelete;
-    }
-}
-
-- (UISwipeActionsConfiguration *)tableView:(UITableView *)tableView trailingSwipeActionsConfigurationForRowAtIndexPath:(NSIndexPath *)indexPath
-{
-    Blog *blog = [self.dataSource blogAtIndexPath:indexPath];
-    NSMutableArray *actions = [NSMutableArray array];
-    __typeof(self) __weak weakSelf = self;
-
-    if ([blog supports:BlogFeatureRemovable]) {
-        UIContextualAction *removeAction = [UIContextualAction
-                                            contextualActionWithStyle:UIContextualActionStyleNormal title:NSLocalizedString(@"Remove", @"Removes a self hosted site from the app")
-                                            handler:^(UIContextualAction * _Nonnull __unused action, __kindof UIView * _Nonnull __unused sourceView, void (^ _Nonnull __unused completionHandler)(BOOL)) {
-            [ReachabilityUtils onAvailableInternetConnectionDo:^{
-                [weakSelf showRemoveSiteAlertForIndexPath:indexPath];
-            }];
-        }];
-
-        removeAction.backgroundColor = [UIColor murielError];
-        [actions addObject:removeAction];
```

Comment:
Does the SwiftUI blog list support removing self-hosted sites?

Reply:
No, but you can remove a site from the homepage.  It's an odd placement. It should probably be in settings like for wp.com sites, but that's a separate issue.

<img width="320px" src="https://github.com/user-attachments/assets/c2c56832-125b-4df1-828f-4fbebc19a061">

I don't know if it was an intentional call to remove this option, but I would vote for keeping it simple and only allowing to removing sites from a single place in the app. I don't think it's a common/quick action that needs to be in the picker. If the goal is to make the app maintainable, we have to keep things simple. @jkmassel, wdyt?

Reply:
I agree about dropping the "delete site" option from the picker, especially since it's kind of expensive/annoying to undo if you made a mistake. 

It's uncommon enough that I think it's totally ok to only have it in the detail view